.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -1,5 +1,5 @@
 ## What? and Why?
-> What does this pull request change/fix? Why was it nessessary?
+> What does this pull request change/fix? Why was it necessary?
```

Comment:
According to Adam all the SDX repo's have the template so it will likely need a card